docs/pages/ViewManager/VM-TemplateTransfer.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Template Transfer
+parent: ViewManager User Guide
+nav_order: 4
+---
+
+# ViewManager
+{: .no_toc }
+Manage all the Views in your project, as well as View Templates and View Range settings in a single window.
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+# Template Transfer tab
+
+The Template Transfer allows you to copy the View Template properties from multiple View Templates.
+
+![Template Transfer](../../../assets\images\ViewManager\VM-Tt-Tab.gif)
+
+<sub>Note: the version on the image may not reflect the [latest version of ViewManager/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+## Select Origin Template
+
+Choose one Origin View Template to copy their properties.
+
+1. On the first table use the first drop-down list to filter view templates by Disciplines.
+
+2. The second drop-down list allows you to filter view templates by View Types.
+
+3. Use the search box to search by view template names.
+
+![Selecting Origin View Template](../../../assets\images\ViewManager\VM-Tt-OriginTemplate.gif)
+
+<sub>Note: the version on the image may not reflect the [latest version of ViewManager/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+### Edit View Template
+
+Right-click on the Origin View Template and select to Edit View Template to open the Revit window, and manage it.
+
+![Edit View Template](../../../assets\images\ViewManager\VM-Tt-EditViewTemplate.gif)
+
+<sub>Note: the version on the image may not reflect the [latest version of ViewManager/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+---
+
+## Select Origin Properties
+
+On this table you are able to choose one or multiple properties from the Origin View Template, to copy to another one.
+
+1. Use the search box to search by view template properties.
+
+2. Use the checkboxes to select properties.
+
+```yaml
+# Note:  
+Check the Hide un-checked properties to see only the choosen properties.
```

Comment:
"Check the "Hide ~~un-checked~~ unchecked properties" to see only the ~~choosen~~ selected properties."